src/browser/modules/Stream/CypherFrame/index.tsx

```diff
@@ -171,7 +171,9 @@ export class CypherFrame extends Component<CypherFrameProps, CypherFrameState> {
     }
 
     // When frame re-use leads to result without visualization
-    if (!this.canShowViz() && this.state.openView === viewTypes.VISUALIZATION) {
+    const doneLoading = this.props.request.status === REQUEST_STATUS_PENDING
```

Comment:
I don't see how status pending equals loading being done, is that true?

Reply:
Oh, you're right. This fix was actually not working.. I just thought it was.